### PR TITLE
DDF-1897: Corrected versioning plugin blueprint

### PR DESCRIPTION
**Pull request by jlcsmith**
#### What does this PR do?
Fixed blueprint specification of implementing interfaces of History Plugin
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@bdeining @djblue 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@pklinef
#### How should this be tested?
Verify versioning plugin bundle starts successfully.

#### What are the relevant tickets?
DDF-1897
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by bdeining** [comment]
:+1: Verified that `catalog-plugin-versioning` starts successfully.

File: catalog/plugin/catalog-plugin-versioning/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <argument ref="catalogFramework" />
```
Comment (bdeining) [inline catalog/plugin/catalog-plugin-versioning/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: We can also remove `xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs"` and associated schemaLocation references from this file

**Comment by jlcsmith** [comment]
Comment addressed.  Should be good to go.

**Comment by pklinef** [comment]
👍

**Pull request merged**